spec/factories/accounting_transactions.rb

```diff
@@ -4,4 +4,10 @@
     qb_transaction_type { Accounting::Transaction::QB_TRANSACTION_TYPES.sample }
     quickbooks_data { {} }
   end
+
+  factory :journal_entry, class: 'Accounting::Transaction' do
```

Comment:
Can you call this `journal_entry_transaction` perhaps? Otherwise it's easily confused with the QB gem JournalEntry model.